__tests__/config/galoy-instances.spec.ts

```diff
@@ -18,6 +18,7 @@ it("get a full object with Custom", () => {
     name: "Custom",
     graphqlUri: "https://api.custom.com/graphql",
     graphqlWsUri: "ws://ws.custom.com/graphql",
+    restUrl: "https://api.custom.com",
```

Comment:
We could call it authUrl. 

I think Rest endpoints will only be limited to auth related task